beetsplug/spotify.py

```diff
@@ -286,8 +287,6 @@ def _get_artist(artists):
             if not artist_id:
                 artist_id = artist['id']
             name = artist['name']
-            # Strip disambiguation number.
-            name = re.sub(r' \(\d+\)$', '', name)
```

Comment:
Removed since I initially copied this logic from Discogs plugin but doesn't apply to Spotify artist strings which don't contain disambiguation numbers.